google/bigtable/bigtable.yaml

```diff
@@ -0,0 +1,34 @@
+# Google Bigtable API service configuration
+
+type: google.api.Service
+config_version: 0
```

Comment:
I just noticed that we're using different config versions all over; pubsub is using version 1, logging is using version 2, and you have version 0 here. Should we settle on one version? (We have only one version of code in toolkit reading these files)